### PR TITLE
Added Netris.ai

### PR DESCRIPTION
**Pull request by ixodie**
Created netris company blurb.

### CODE REVIEW

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Comment by dougaitken** [comment]
Hi @ixodie can you delete #1135 and add the profile to the main readme file as part of this PR, please?

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Comment by ixodie** [comment]
Sorry for the mixup, both file changes have been merged into one PR.

File: company-profiles/netris.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Netris develops and supports a network management platform, providing Automatic NetOps.  This simplifies infrastructure deployments for data centers and edge compute environments.
+
+## Company size
+
+20+ 
+
+## Remote status
+
+Posibility to remote work - Get things done - from home or an office.
```
Comment (dougaitken) [inline company-profiles/netris.md:13]: Spelling error - `possibility`

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Review by ixodie — COMMENTED** [review]
Updated.

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Review by dougaitken — APPROVED** [review]

**Pull request merged**